quiz.sql

```diff
@@ -7,29 +7,69 @@ CREATE DATABASE westeros;
 --   `people` has an id ( serial primary key), name (text), title (text), quote (text), image (text) and age (integer)
 
 -- your code starts here
+CREATE TABLE houses (
+  id serial primary key,
+  name text,
+  motto text,
+  sigil text
+);
+
+CREATE TABLE people (
+  id serial primary key,
+  name text,
+  title text,
+  quote text,
+  image text,
+  age integer
+);
 
 -- your code ends here
 
 -- Question 2. Insert the data from `westeros/houses.csv` into the `houses` table and the data from `westeros/people.csv` into the `people` table.
 
 -- your code starts here
+INSERT INTO houses(name, motto, sigil) values('Baratheon', 'Ours is the fury.', 'http://i.imgur.com/iJtBKIR.png');
+INSERT INTO houses(name, motto, sigil) values('Bolton', 'Our blades are sharp.', 'http://i.imgur.com/uPoc9Jh.jpg');
+INSERT INTO houses(name, motto, sigil) values('Lannister', 'Hear me roar!', 'http://i.imgur.com/gOwmzoU.jpg');
+INSERT INTO houses(name, motto, sigil) values('Martell', 'Unbowed, unbent, unbroken.', 'http://i.imgur.com/XaIiVNI.jpg');
+INSERT INTO houses(name, motto, sigil) values('Stark', 'Winer is coming.', 'http://i.imgur.com/zpVjXsa.jpg');
+INSERT INTO houses(name, motto, sigil) values('Targaryen', 'Blood and fire.', 'http://i.imgur.com/4FuSpZt.jpg');
+
+
+INSERT INTO people(name, title, quote, image, age) values('Denaryus Targaryen', 'Mother of Dragons', 'I will take what is mine', 'http://i.imgur.com/DjHJq4E.jpg', 18);
+INSERT INTO people(name, title, quote, image, age) values('Jon Snow', 'Commander of Nigh watch', 'Night gathers', 'http://i.imgur.com/WadV6VI.jpg', 19);
+INSERT INTO people(name, title, quote, image, age) values('Tyrion Lannister', 'Hand of the King', 'Always pays his debts', 'http://i.imgur.com/rNHNWca.jpg', 35);
+INSERT INTO people(name, title, quote, image, age) values('Arya Stark', 'Princess of Winterfell', 'Anyone can be killed', 'http://i.imgur.com/Csq9exg.jpg', 14);
+--etc..
+
 
 -- your code ends here
 
 -- Question 3. Write a query to get all attributes of people greater than a certain age *or* having a specific title.
 
 -- your code starts here
+SELECT name, title, quote, image, age
```

Comment:
You can use '*' in place of a list of all of the attributes here. Also, you can combine both of these queries into one like so:

`SELECT * FROM people WHERE age > 2 OR name = 'Princess of Winterfell';`